studies/models.py

```diff
@@ -249,6 +262,22 @@ def add_study_created_log(sender, instance, created, **kwargs):
             user=instance.creator
         )
 
+@receiver(post_save, sender=Study)
+def check_modification_of_approved_study(sender, instance, created, **kwargs):
+    """
+    Puts study back in "rejected" state if study is modified after it's already been approved.
+    Leaves comment for user with explanation.
+    """
+    approved_states = ['approved', 'active', 'paused', 'deactived']
```

Comment:
I worry about going around the workflow engine here but I also can't think of a better way to do it. 

I guess we could add another trigger that can only be used by the system. What do you think?

We should also probably warn the user, prior to saving, that their study will be deactivated and they will have to wait to be re-approved.